src/services/email-line-item-delivery.js

```diff
@@ -28,93 +29,105 @@ module.exports = {
    * @param {Date} params.date
    * @param {object} [params.imageOptions] Imgix options to use in the creative image url.
    * @param {object} [params.advertiserLogoOptions] Imgix options to use in the creative image url.
+   * @param {object} [params.opts] Generic options see campaign-delivery.js for potential examples
    */
   async findFor({
     placementId,
     date,
     imageOptions,
     advertiserLogoOptions,
+    opts,
   } = {}) {
-    const [placement, lineItem] = await Promise.all([
+    const limit = opts.n ? parseLimit(opts.n) : 1;
+    const [placement, lineItems] = await Promise.all([
       this.getPlacement({ placementId }),
-      this.getLineItemFor({ placementId, date }),
+      this.getLineItemsFor({ placementId, date }),
```

Comment:
This should pass `limit`, there's no reason to query for more line items than needed

Reply:
https://github.com/parameter1/fortnight-graph/blob/master/src/services/campaign-delivery.js#L182

Corrected, I've provided above where I originally got this from as a point a reference however.